store/sqlstore/upgrade.go

```diff
@@ -1011,13 +1011,14 @@ func upgradeDatabaseToVersion535(sqlStore *SqlStore) {
 
 	sqlStore.CreateColumnIfNotExists("SidebarCategories", "Collapsed", "tinyint(1)", "boolean", "0")
 
-	sqlStore.CreateColumnIfNotExistsNoDefault("Channels", "TotalMsgCountRoot", "bigint", "bigint")
+	sqlStore.CreateColumnIfNotExists("Channels", "TotalMsgCountRoot", "bigint", "bigint", "0")
```

Comment:
Adding a column with a default value for PG versions < 11 will obtain an `ACCESS EXCLUSIVE` lock right? For customers that have big channels table the default value will slow down their boot up.

Reply:
@nronas - I think that's a possibility. We did however add a release-note saying that migration can take up to a few minutes. cc @streamer45

Reply:
Oh superb.